app/controllers/api/v2/search_controller.rb

```diff
@@ -8,6 +8,13 @@ class Api::V2::SearchController < Api::BaseController
   before_action -> { authorize_if_got_token! :read, :'read:search' }
   before_action :validate_search_params!
 
+  with_options if: :user_signed_in? do
```

Comment:
Did you mean `unless:`?

Reply:
Yes, it should be! Pushed commit to add coverage that would have caught that switch, and restored the correct condition there.